clients/roscpp/include/ros/rosout_appender.h

```diff
@@ -35,10 +35,11 @@
 #ifndef ROSCPP_ROSOUT_APPENDER_H
 #define ROSCPP_ROSOUT_APPENDER_H
 
+#include <ros/common.h>
 #include <ros/message_forward.h>
-#include "common.h"
```

Comment:
Nitpick: I assume this change isn't necessary and can be removed?

Reply:
I think it was a leftover from a previous version, where I modified `common.h.in` to  set the `BOOST_THREAD_HAS_CONDATTR_SET_CLOCK_MONOTONIC` flag there, which was a really bad idea because then the dangerous macro also affects user packages that include this header.